gatsby/gatsby-config.js

```diff
@@ -5,10 +5,20 @@ require("dotenv").config({
 	path: `.env.${process.env.NODE_ENV}`,
 });
 
+/** The date from which to get updates */
+const changesSinceDate = process.env.CHANGES_SINCE
+	? moment.utc(process.env.CHANGES_SINCE).toDate()
+	: moment()
+			.utc()
+			.subtract(1, "months")
+			.startOf("month")
+			.toDate();
+
 module.exports = {
 	siteMetadata: {
 		siteUrl: "https://cks.nice.org.uk",
```

Comment:
Hmm it's not that simple though as this `siteUrl` parameter is used at build time, so we'd have to variable replace in the resulting files e.g. sitemap.xml rather than gatsby-config

Reply:
Also, I didn't change this in this PR so am resolving this....